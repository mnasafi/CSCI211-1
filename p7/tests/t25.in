echo t25: Insert months into tree, then call balanced, rebalance, balanced, print
echo t25: This is different from previous rebalance tests because there are an even number of elements
insert January
insert February
insert March
insert April
insert May
insert June
insert July
insert August
insert September
insert October
insert November
insert December
breadth
balanced
echo calling rebalance
rebalance
breadth
balanced
