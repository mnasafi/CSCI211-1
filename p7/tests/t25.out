t25: Insert months into tree, then call balanced, rebalance, balanced, print
t25: This is different from previous rebalance tests because there are an even number of elements
{January, February, March, April, June, May, August, July, September, December, October, November}
Tree is not balanced.
calling rebalance
{July, December, May, April, February, June, October, August, January, March, November, September}
Tree is balanced.
